roles/openshift_node/tasks/main.yml

```diff
@@ -77,7 +77,7 @@
 - name: Check scheduleable state
   delegate_to: "{{ openshift_first_master }}"
   command: >
-    {{ openshift.common.client_binary }} get node {{ inventory_hostname }}
+    {{ openshift.common.client_binary }} get node {{ openshift_hostname }}
```

Comment:
It would probably be better to take a stance that `openshift_hostname` should always be set for Master and Node roles.